project/plugins.sbt

```diff
@@ -1,6 +1,5 @@
-addSbtPlugin("edu.gemini"     % "sbt-lucuma"        % "0.4.2")
+resolvers += Resolver.sonatypeRepo("snapshots")
+addSbtPlugin("edu.gemini"     % "sbt-lucuma-lib"    % "0.6-3ce8fc2-SNAPSHOT")
```

Comment:
Should we wait for a release?

Reply:
Yes, no rush to merge. I can transition all the builds using the snapshot, then we can cut 0.6.0 final.